CHANGELOG.md

```diff
@@ -4,10 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] - 2023-03-28
+## [0.20.1] - 2023-04-07
 
 ### Added
-* Replaced llvm_spirv from oneAPI path by dpcpp-llvm-spirv package.
+* Replaced llvm_spirv from oneAPI path by dpcpp-llvm-spirv package.(#979)
+* Added Dockerfile and a manual workflow to publish pre-built packages to the repo.(#973)
+
+### Changed
+* Allowed shape to be a positional only argument when constructing a tensor. (#993)
```

Comment:
#993 should go under fixed. (Fixed default dtype derivation when creating a dpnp.ndarray)

Reply:
> Reference in new

Moved.